src/masternodes/rpc_accounts.cpp

```diff
@@ -0,0 +1,1429 @@
+#include <masternodes/mn_rpc.h>
+
+std::string tokenAmountString(CTokenAmount const& amount) {
+    const auto token = pcustomcsview->GetToken(amount.nTokenId);
+    const auto valueString = strprintf("%d.%08d", amount.nValue / COIN, amount.nValue % COIN);
+    return valueString + "@" + token->symbol + (token->IsDAT() ? "" : "#" + amount.nTokenId.ToString());
+}
+
+UniValue AmountsToJSON(TAmounts const & diffs) {
+    UniValue obj(UniValue::VARR);
+
+    for (auto const & diff : diffs) {
+        auto token = pcustomcsview->GetToken(diff.first);
+        auto const tokenIdStr = token->CreateSymbolKey(diff.first);
+        obj.push_back(ValueFromAmount(diff.second).getValStr() + "@" + tokenIdStr);
```

Comment:
While we are here. That's same thing, 
```
for (auto const & diff : diffs) {
    obj.push_back(tokenAmountString({diff.first, diff.second}));
}
```
in `tokenAmountString`
```
auto const tokenIdStr = token->CreateSymbolKey(amount.nTokenId);
return ValueFromAmount(diff.nValue).getValStr() + "@" + tokenIdStr;
```

Reply:
Do you mean to make it like this?

`std::string tokenAmountString(CTokenAmount const& amount) {
    auto const tokenIdStr = token->CreateSymbolKey(amount.nTokenId);
    return ValueFromAmount(diff.nValue).getValStr() + "@" + tokenIdStr;
}
UniValue AmountsToJSON(TAmounts const & diffs) {
    UniValue obj(UniValue::VARR);
    for (auto const & diff : diffs) {
        obj.push_back(tokenAmountString({diff.first, diff.second}));
    }
    return obj;
}`

Reply:
Yep, you can verify they returns same string in rpc functional test (i mean not to write one, just to make sure they are identical, in existing one that uses these functions)

Reply:
Not exactly the same, but as I see from tests it doesn't use tokenAmountString as it uses different output if indexed_amounts true.